packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.html

```diff
@@ -1,7 +1,9 @@
 <template>
   <template if:true={shouldDisplay}>
-    <li class={tabClass} title={label} role="presentation">
-      <a class="slds-tabs_default__link" role="tab" onclick={select}>{label}</a>
-    </li>
+    <div tabindex="0" role="tab" onkeydown={onKeyDown}>
```

Comment:
I'm not sure `role="tab"`  is appropriate here. We're using the term "Tab" describe this and I think the concept is close enough. But given each tab doesn't actually have an associate tab panel that it displays when selected, I think it might be misleading. They're really buttons that apply a filter. 😅

Curious to know other thoughts on this.

Reply:
I agree, role="tab" is for a control of type tabset that we don't want to be using.
https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-manual.html
It was reflected in the assessment that this "looks" like a tabset but isn't really a tabset, so I would feel better if those were just buttons.

Reply:
changed to button